.nocover.yaml

```diff
@@ -21,9 +21,9 @@ nocover_regexes:
   # generation/InfoMapping.py
   # https://github.com/coala/coala-quickstart/issues/275
   - INFO_SETTING_MAPS =
+  - def language_percentage
   # generation/Project.py
   # https://github.com/coala/coala-quickstart/issues/276
-  - def language_percentage
```

Comment:
this line shouldn't be moved, as the commented issue link directly above this refers to it

Reply:
no you have moved the line to someplace else instead of removing it

Reply:
ahhh!  thx!